app/styles/_global.scss

```diff
@@ -3,3 +3,48 @@
 #toast-container > div { // stylelint-disable-line selector-max-id
     opacity: 1;
 }
+
+@media (max-width: 480px) {
+    :root {
+        --ratio: 1.2;
+    }
+}
+@media (min-width: 480px) and (max-width: 768px) {
+    :root {
+        --ratio: 1.414;
+    }
+}
+@media (min-width: 768px) {
+    :root {
+        --ratio: 1.5;
+    }
+}
+
+:root {
+    --lineHeight: 1.5;
+    --baseFontSize: 112.5%;
+    --stepUp1: calc(1em * var(--ratio));
```

Comment:
Should this be `1rem`, to guarantee it's actually based off `baseFontSize`?

Reply:
This is a lot of "building off the research of others", but according to https://stevenloria.com/responsive-typography/ (which is based off of https://zellwk.com/blog/media-query-units/ ), "Use em units for media queries because they behave the most consistently across browsers." My understanding is that the scaling will work with any system, it's just *how* it works will be different, depending. 

Was there something specific that you're pretty sure I'm misunderstanding, because what I don't know about this is extensive.

Reply:
My concern with `em` is that a heading's font-size will be calculated based on its parent element's font-size, which could be inconsistent on one page. e.g.
```html
<h2>Section!</h2>
<div style="font-size: 50px;">
  <h3>Subsection!</h3>
</div>
```
the <h3> will be larger than the <h2>. Granted, we should never do that sort of thing, but I feel like there'd be peace of mind knowing that headings will always have the same font-size, no matter where they live.

Reply:
So would you prefer handling a thing that we really should never do, or having Safari behave the same as Firefox, Chrome, and IE do near media-size breaks? That's basically the trade-off; Safari doesn't do what everyone else does.

Reply:
...on the other hand, that might be a deliberate feature. Maybe it's preferred to be able to do
```html
<div style="font-size:5px">
  <h2>tiny headings</h2>
  <p>and correspondingly tiny text!</p>
</div>
<div style="font-size:50px">
  <h2>BIG headings</h2>
  <p>and BIG text!</p>
</div>
```
so the headings are always proportional to the text around them. I don't know, I guess I'm fine either way.